compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -219,9 +224,19 @@ void addSimplePackBasedPassPipeline(OpPassManager &pm,
   modulePassManager.addNestedPass<func::FuncOp>(
       createAMDAIEBufferizeToAllocationPass(bufferizeOptions2));
 
+  {
+    AMDAIELowerToUKernelsOptions options;
+    options.passPipeline = AIEPassPipeline::SimplePackPipeline;
+    options.pathToUkernels = clPathToUkernels;
+    modulePassManager.addNestedPass<func::FuncOp>(
+        createAMDAIELowerToUKernelsPass(options));
+  }
   // Comprehensive bufferization
   addAMDAIEBufferizePasses(modulePassManager);
+  modulePassManager.addPass(createLowerUKernelOpsToCallsPass());
   modulePassManager.addPass(memref::createFoldMemRefAliasOpsPass());
+
+  modulePassManager.addPass(createLowerUKernelOpsToCallsPass());
```

Comment:
What is this pass for? Why do you call this pass twice?

Reply:
The pass is used for this conversion: `iree_codegen.ukernel @myfunc` -> `func.call @myfunc`.

The second invocation was a mistake - it didn't affect the flow because the first invocation changed all `iree_codegen.ukernel` to `func.call` - thanks for catching - I've updated removed it.

Reply:
Looks good to me. I'll leave for Mahesh to take a look also.